### DISCUSSION TITLE
VLESS+Reality证书问题

### DISCUSSION THREAD

**Discussion by HShiDianLu** [XTLS/Xray-core#3240] [Q&A]
Reality正常配置是不需要证书的，但我之前为我的域名申请了一个ssl证书，现在导致直接访问该域名报`NET::ERR_CERT_COMMON_NAME_INVALID`（服务器将来还要有其他服务），配置nginx反代挂证书的话443端口是占用的，有什么其他方式解决吗？

**Comment by chika0801** [comment]
https://github.com/chika0801/Xray-examples/tree/main/VLESS-Vision-REALITY/steal_oneself

尝试理解这套配置
